tests/e2e-leg-6/revivedb-1-node-restore/15-errors.yaml

```diff
@@ -0,0 +1,18 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# No jobs should be created since the EventTrigger refers to a vdb that doesn't
```

Comment:
I don't think we are creating EventTrigger's in this PR. Maybe a copy/paste from the testcase you based this one on? I suggest we remove this errors file.

Reply:
will do